docs/resources/dhcp_fixed_address.md

```diff
@@ -69,6 +69,14 @@ resource "bloxone_dhcp_fixed_address" "example_fixed_address_na" {
   tags = {
     location : "site1"
   }
+  dhcp_options = [
+    {
+      description  = "Option 1"
+      option_code  = "234"
+      option_value = "true"
+      type         = "boolean"
+    }
```

Comment:
I don't think this will work. `description` is not a field part of dhcp_options. Also `option_code` is an UUID for the dhcp_option_code object and not just the code. And `type` here is either `option` or `group`.

Reply:
done